src/main/java/com/xceptance/neodymium/util/LighthouseUtils.java

```diff
@@ -0,0 +1,217 @@
+        });
+        
+        // validate jsonpaths in neodymium properties
+        validateAudits(jsonFile);
+        
+        // add report html to allure
+        Allure.addAttachment(reportName, "text/html", FileUtils.openInputStream(new File("target/" + reportName + ".report.html")), "html");
+        
+        // switch back to saved URL
+        driver.switchTo().window(newWindow);
+        driver.get(URL);
+    }
+    
+    /**
+     * <p>
+     * Opens a new tab apart from the first tab with the test automation, adds a handle and closes the first tab.
+     * If the first tab with the test automation is not closed, the Lighthouse report will not have proper values,
+     * because it will interfere with the Lighthouse report generation.
+     * </p>
+     * 
+     * @param driver
+     *            The current webdriver
+     * @return A new empty tab with a window handle
+     */
+    private static String windowOperations(WebDriver driver)
+    {
+        String originalWindow = driver.getWindowHandle();
+        driver.switchTo().newWindow(WindowType.TAB);
+        String newWindow = driver.getWindowHandle();
+        driver.switchTo().window(originalWindow);
+        driver.close();
+        return newWindow;
+    }
+
+    /**
+     * <p>
+     * Uses <a href="https://developer.chrome.com/docs/lighthouse/overview?hl=de">Lighthouse</a> (Copyright Google) 
+     * to create a Lighthouse report of the current URL.
+     * </p>
+     * 
+     * @param URL
+     *            The current URL the Lighthouse report should be generated on
+     * @param reportName
+     *            The name of the Lighthouse report attachment in the Allure report 
+     * @throws Exception 
+     */
+    private static void lighthouseAudit(String URL, String reportName) throws Exception
+    {
+        ProcessBuilder builder = new ProcessBuilder();
+        
+        if (System.getProperty("os.name").toLowerCase().contains("win")) 
+        {
+            builder = new ProcessBuilder("cmd.exe", "/c", Neodymium.configuration().lighthouseBinaryPath(), "--chrome-flags=\"--ignore-certificate-errors\"", URL, "--port=" + Neodymium.getRemoteDebuggingPort(), "--preset=desktop", "--output=json", "--output=html", "--output-path=target/" + reportName + ".json");
+        }
+        else if (System.getProperty("os.name").toLowerCase().contains("linux") || System.getProperty("os.name").toLowerCase().contains("mac"))
+        {
+            builder = new ProcessBuilder(Neodymium.configuration().lighthouseBinaryPath(), "--chrome-flags=\"--ignore-certificate-errors\"", URL, "--port=" + Neodymium.getRemoteDebuggingPort(), "--preset=desktop", "--output=json", "--output=html", "--output-path=target/" + reportName + ".json");
```

Comment:
for linux please change this to call the shell by using `new ProcessBuilder("sh", "-c", Neodymium.config....`